test/config_test/examples/mnist-annotation.test.yml

```diff
@@ -3,6 +3,7 @@ experimentName: default_test
 maxExecDuration: 5m
 maxTrialNum: 2
 trialConcurrency: 1
+nniManagerIp: 52.168.146.89
```

Comment:
Please do not write nniManagerIp in *.test.yml,  write it in training_service.yml

Reply:
moved to training_service.yml